samples/snippets/bqml_getting_started_test.py

```diff
@@ -0,0 +1,88 @@
+
+    # Start by selecting the data you'll use for training. `read_gbq` accepts
+    # either a SQL query or a table ID. Since this example selects from multiple
+    # tables via a wildcard, use SQL to define this data. Watch issue
+    # https://github.com/googleapis/python-bigquery-dataframes/issues/169
+    # for updates to `read_gbq` to support wildcard tables.
+
+    df = bpd.read_gbq(
+        """
+        -- Since the order of rows isn't useful for the model training,
+        -- generate a random ID to use as the index for the DataFrame.
+        SELECT GENERATE_UUID() AS rowindex, *
+        FROM
+        `bigquery-public-data.google_analytics_sample.ga_sessions_*`
+        WHERE
+        _TABLE_SUFFIX BETWEEN '20160801' AND '20170630'
+        """,
+        index_col="rowindex",
+    )
+
+    # Extract the total number of transactions within
+    # the Google Analytics session.
+    #
+    # Because the totals column is a STRUCT data type, call
+    # Series.struct.field("transactions") to extract the transactions field.
+    # See the reference documentation below:
+    # https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.operations.structs.StructAccessor#bigframes_operations_structs_StructAccessor_field
+    transactions = df["totals"].struct.field("transactions")
+
+    # The "label" values represent the outcome of the model's
+    # prediction. In this case, the model predicts if there are any
+    # ecommerce transactions within the Google Analytics session.
+    # If the number of transactions is NULL, the value in the label
+    # column is set to 0. Otherwise, it is set to 1.
+    label = transactions.notnull().map({True: 1, False: 0})
+
+    # Extract the operating system of the visitor's device.
+    operatingSystem = df["device"].struct.field("operatingSystem")
+    operatingSystem = operatingSystem.fillna("")
+
+    # Extract whether the visitor's device is a mobile device.
+    isMobile = df["device"].struct.field("isMobile")
+
+    # Extract the country from which the sessions originated, based on the IP address.
+    country = df["geoNetwork"].struct.field("country").fillna("")
+
+    # Extract the total number of page views within the session.
+    pageviews = df["totals"].struct.field("pageviews").fillna(0)
+
+    # Combine all the feature columns into a single DataFrame
+    # to use as training data.
+    features = bpd.DataFrame(
+        {
+            "os": operatingSystem,
+            "is_mobile": isMobile,
+            "country": country,
+            "pageviews": pageviews,
+        }
+    )
+
+    # Logistic Regression model splits data into two classes, giving the
+    # a confidence score that the data is in one of the classes.
+    model = LogisticRegression()
+    model.fit(features, label)
+
+    # The model.fit() call above created a temporary model.
+    # Use the to_gbq() method to write to a permanent location.
+    model.to_gbq("bqml_tutorial.sample_model", replace=True)
```

Comment:
FYI: We're getting

```
E           google.api_core.exceptions.BadRequest: 400 Concurrent update on same model: bigframes-dev:bqml_tutorial.sample_model is not supported. Share your usecase with the BigQuery DataFrames team at the [https://bit.ly/bigframes-feedback](https://www.google.com/url?q=https://bit.ly/bigframes-feedback&sa=D) survey.
```

failure in our test suite: https://fusion2.corp.google.com/invocations/8a7513c8-e7c9-4b5b-82fe-9a83c176fbc1/targets/bigframes%2Fpresubmit%2Fe2e/log

I think we'll need a test fixture for this to create a temporary place for the model and clean it up when the test finishes.

1. Create a file called `samples/snippets/conftest.py`.
2. In the `conftest.py` file you create, add a fixture called `random_model_id`, similar to this one: https://github.com/googleapis/python-bigquery/blob/f804d639fe95bef5d083afe1246d756321128b05/samples/snippets/conftest.py#L101-L111 except it'll call `delete_model(...)` instead of `delete_table(...)`.

    You'll also need to add "prefixer" https://github.com/googleapis/python-bigquery/blob/f804d639fe95bef5d083afe1246d756321128b05/samples/snippets/conftest.py#L21 and `bigquery_client` fixture https://github.com/googleapis/python-bigquery/blob/f804d639fe95bef5d083afe1246d756321128b05/samples/snippets/conftest.py#L33-L36
3. Update your code sample to use the new `random_model_id` fixture.

    Look how we do it in the remote functions test: https://github.com/googleapis/python-bigquery-dataframes/blob/81125f9505ad98e89939769a8e1fcf30518705f0/samples/snippets/remote_function.py#L17-L23 but instead you'll be setting `your_model_id = random_model_id` and calling

    ```
    model.to_gbq(
        your_model_id,  # "project.dataset.model_id" or "dataset.model_id"
        replace=True,
    )
    ```